### ISSUE TITLE
wget certificate issues while running script in WSL

### ISSUE THREAD

**Issue by fasteddy516** [myspaghetti/macos-guest-virtualbox#78]
Running in WSL/Debian on Windows 10, I kept getting *couldn't download catalog* errors after selecting the macOS version (H/M/C) with no indication as to what the issue was.  I edited the script to remove the `--quiet` option from wget, and found that wget was exiting with the following errors:
```
ERROR: The certificate of ‘swscan.apple.com’ is not trusted.
ERROR: The certificate of ‘swscan.apple.com’ hasn't got a known issuer.
```
If there is a way to detect/correct this automatically in the script that would be ideal.  I used the nasty wget `--no-check-certificate` option just to keep things moving while I spend some time looking up how to trust certificates in Linux/WSL, but that's more of a completely-unsecure cheat than a solution.

**Comment by myspaghetti** [comment]
Please check if [some antivirus software is blocking the https request](https://github.com/Microsoft/WSL/issues/3761).

Additionally, it would be helpful if you could run the following and post the output:
```
wget --debug 'https://swscan.apple.com/content/catalogs/others/index-10.14-10.13-10.12-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard.merged-1.sucatalog'
```

**Comment by myspaghetti** [comment]
Please check if [some antivirus software is blocking the https request](https://github.com/Microsoft/WSL/issues/3761).

Additionally, it would be helpful if you could run the following and post the resulting `wget.log` file: 
```
wget --debug -o wget.log -O /dev/null 'https://swscan.apple.com/content/catalogs/others/index-10.14-10.13-10.12-10.11-10.10-10.9-mountainlion-lion-snowleopard-leopard.merged-1.sucatalog'
```

**Comment by fasteddy516** [comment]
I checked the antivirus thread you mentioned - the only thing running is Windows Defender, which doesn't seem to be listed as a culprit.

I've posted two logs:
* [wget.log](https://github.com/myspaghetti/macos-guest-virtualbox/files/3356326/wget.log) - wget fails due to certificate error.

* [wget_nocheck.log](https://github.com/myspaghetti/macos-guest-virtualbox/files/3356327/wget_nocheck.log) - wget succeeds because I added the `--no-check-certificate` option to the command.

For what it's worth, I used the older version of your script (commit 6342cec292b167488b7c984d4407893711b74ac6)  a couple of weeks ago without issues - not sure if this issue stems from script changes since then or windows updates.

**Comment by myspaghetti** [comment]
From the log file:

`Certificates loaded: 0`

Either you have no certificates installed on WSL, or they're not listed in the `wget` configuration files. Try configuring `wget` to point to the certificate store, or try reinstalling `openssl` and `ca-certificates`. Since this is outside the scope of the script I will be closing the bug as "invalid". (Plus your `.wget-hsts` file is world-writable so you might want to `chmod 600` it.)

**Comment by fasteddy516** [comment]
Thank you - installing `ca-certificates` fixed the problem.

FWIW, I installed WSL and Debian (via the Windows Store) for the sole purpose of running this script.  I ran `sudo apt-get update` and `sudo apt-get install coreutils unzip wget dmg2img`, then ran the script.  I'm not sure if you want to mention `ca-certificates` in the `readme` (or check if the package is installed in the script as is done with the others), but it was the single missing piece that kept me from running the script as-is in a fresh WSL/Debian install.
